### PR TITLE
feat: header,section anf footer added

### PR DESCRIPTION
**Pull request by mihrab34**

### CODE REVIEW

**Review by imhta — CHANGES_REQUESTED** [review]
# Changes required 🛑 
@mihrab34 you have done a good job but still there is few changes to address.

- In mobile version, the order of sections should be `Top Story`, `More Stories`, `Featured Stories` and then `Opinion`.
- In mobile version(Pixel 2 XL), There is a issue with navbar and sign in button. (To recreate this issue kindly open the website in chrome then `Ctrl`+`Shift`+`M`, then select Pixel 2 XL in devices list) 
image

- In mobile and tablet version, page is creating a horizontal scrolling issue, kindly check that. I think there is a issue with the sizing of the container.
image
#### Don't Forget to make another code review.
## Good luck
**Pon Muthu Selvam N**
Feel free contact me 
[Github //@imhta](https://github.com/imhta) | [Twitter //@nponmuthuselvam](https://twitter.com/ponmuthuselvam)

**Comment by shshamim63** [comment]
## Status :arrow_forward:  Changes Required :arrow_backward: 
___
Hi @mihrab34 :wave: ,
Awesome job on this project :clap: . I can see most of the items are rendering on the place that it should. Code looks nice and clean. But some issues are actually holding me to approve this project. I am sure that, you will cover them easily.
___
#### Please follow the instructions below :point_down:  to complete the project.
___
### :clubs: Mandatory Changes :clubs: 
___
1. On the mobile view the nav item is breaking down. Which is actually catching my eye. Inspect the element and you will get the way to fix it.
news1
2. On the header-top class at your css line no. 428, I can see a !important. Is it possible to remove it?
___
### :spades: Optional Changes :spades: 
___
1. On tab view the marked section looks too big. I think if you make this section smaller it will look more beautiful.
news2
___
### Please follow the instructions above :point_up: and send another code review request.
___
## :herb:  Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1146 @@
...
+              <div class="headline"><a href="#">ISIS Spy Who Betrayed Baghdadi Was Under Pressure From Family: SDF
+                  Chief</a>
+              </div>
+              <p class="description">
+                The unnamed informant's family reportedly suffered under Islamic State rule prompting him to seek
+                revenge,
+                General Mazloum Abdi said.
+              </p>
+            </div>
+          </div>
```
Comment (suretrust) [inline index.html:611]: On medium screen, the editor's picks section should have 2 columns. Please refer to the images below:

editor
editor2

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @mihrab34,

### Great job thus far on the project. :raised_hands:  Just a few suggestions to wrap it up:

- [ ] Please see my comment on your code.

- [ ] You used Git flow as opposed to the GitHub flow requirement. See [this article](https://www.freshconsulting.com/git-development-workflows-git-flow-vs-github-flow/) to understand the difference.

- [ ] In mobile view, let's arrange the sections in the correct order, `Top story` => `More stories` => `Culture and travel` => `Featured stories` => `Opinion` => `Latest news`. You can achieve this using bootstrap's `order` class.

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Pull request merged**